### ISSUE TITLE
Polymorphism section

### ISSUE THREAD

**Issue by MissMeg** [MissMeg/oop-for-beginners#7]
Explain what polymorphism is with a simple example

Master List: #3

**Comment by rithin-jose** [comment]
Polymorphism is the ability of an object to take on many forms. In laymans language we can say that its is the ability to use the same thing for different purposes.
#!/usr/bin/env python

class Human:

    def sayHello(self, name=None):
    
        if name is not None:
            print('Hello ' + name)
        else:
            print('Hello ')
        

# Create instance
obj = Human()
    
# Call the method
obj.sayHello()
    
# Call the method with a parameter
obj.sayHello('Guido')



OUTPUT
Hello
Hello Guido
